core/cap-0046-06.md

```diff
@@ -198,20 +205,27 @@ compliance functionality.
 * Admin Interface
 *
 \******************************************************************************/
-// If "admin" is the administrator, burn "amount" from "from"
-fn burn(admin: Signature, nonce: BigInt, from: Identifier, amount: BigInt) -> Result<(), Error>;
+// If "admin" is the administrator and clawback is not disabled, clawback "amount" from "from".
+// The "amount" is burned. This function can be used on deauthorized balances. 
+fn clawback(admin: Signature, nonce: BigInt, from: Identifier, amount: u128) -> Result<(), Error>;
+
+// If "admin" is the administrator, deauthorize "id"
+fn deauthorize(admin: Signature, nonce: BigInt, id: Identifier) -> Result<(), Error>;
 
-// If "admin" is the administrator, freeze "id"
-fn freeze(admin: Signature, nonce: BigInt, id: Identifier) -> Result<(), Error>;
+// If "admin" is the administrator, authorize "id"
```

Comment:
I think we should explain what does 'authorize' actually mean. Normally you authorize someone to do something, while here we just say 'authorize'.